### PR TITLE
bump version to 1.5.0

### PR DESCRIPTION
**Pull request by bvered**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/IBM/ibm-storage-odf-block-driver
 go 1.18
 
 require (
-	github.com/IBM/ibm-storage-odf-operator v1.4.0
+	github.com/IBM/ibm-storage-odf-operator v1.3.1-0.20230821004148-24b0497905fe
```
Comment (kasserater) [inline go.mod:6]: how did the version go backwards?

File: go.sum [hunk 2]
```diff
@@ -45,8 +45,8 @@ github.com/Azure/go-autorest/logger v0.2.1/go.mod h1:T9E3cAhj2VqvPOtCYAvby9aBXkZ
 github.com/Azure/go-autorest/tracing v0.6.0/go.mod h1:+vhtPC754Xsa23ID7GlGsrdKBpUA79WCAKPPZVC2DeU=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
-github.com/IBM/ibm-storage-odf-operator v1.4.0 h1:OXE+umdZtrdTV31+v0dEYkst5LexxrFJUkGqv+iBCB4=
-github.com/IBM/ibm-storage-odf-operator v1.4.0/go.mod h1:TmOsuYoZIu70IMMZzdxfj1OH5EPPon3u1qzuQO9Xaxo=
+github.com/IBM/ibm-storage-odf-operator v1.3.1-0.20230821004148-24b0497905fe h1:6gonpX4fEfwlOyc8O603vXtuseTLg8Er7UTZGgWsEio=
```
Comment (kasserater) [inline go.sum:48]: how did the version go backwards?

File: go.mod (hunk 1 above)
Comment (bvered) [reply go.mod:6]: the first (old line) 1.4.0 is a tag 
the second (new line) is a commit number go is calculating and generating on it's own.
It didn't go backwards but that's how go decided to call this tag... we don't have impact on it. 

On official release we will always put the official tag and not a commit / branch name

File: go.sum (hunk 2 above)
Comment (bvered) [reply go.sum:48]: the first (old line) 1.4.0 is a tag 
the second (new line) is a commit number go is calculating and generating on it's own.
It didn't go backwards but that's how go decided to call this tag... we don't have impact on it. 

On official release we will always put the official tag and not a commit / branch name

**Review by kasserater — APPROVED** [review]
LGTM

**Pull request closed without merging**